src/cljam/io/util/bin.clj

```diff
@@ -1,44 +1,45 @@
 (ns cljam.io.util.bin
-  (:require [cljam.io.util.chunk :as util-chunk]))
+  (:require [cljam.io.util.chunk :as util-chunk])
+  (:import [cljam.io.util.chunk Chunk]))
```

Comment:
Unused imports here. Using [clj-kondo](https://github.com/borkdude/clj-kondo/blob/master/doc/editor-integration.md), we can catch that.